### DISCUSSION TITLE
[Feature Request] Support for Base64 and ROT13 encoded chats

### DISCUSSION THREAD

**Discussion by adamrb** [danny-avila/LibreChat#37] [Feature Requests & Suggestions]
In testing this user script: https://greasyfork.org/en/scripts/461220-powerful-bing-chat-ai I found that Bing AI is less likely to refuse to answer when the chat is encoded in Base64 or ROT13. Supporting this on the client side, especially for "jailbroken" engines could provide an additional layer of opportunities for bypassing restrictions.

**Comment by DavesDevFails** [comment]
Base64 does seem to bypass a lot of the filtering, however it also seems to degrade the quality of the responses somewhat. Would be great to have as an option definitely.

**Comment by danny-avila** [comment]
Not too sure about this one. People are getting banned from Bing [for using this](https://www.aitesters.org/d/7-the-account-banning-of-bing-discuss-here-bing/10) and the link you provided also mentions ```The team is banning accounts, there is a High-Risk of using this tool now.```
